modules/vsphere-problem-detector-config-checks.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * storage/vsphere-problem-detector-operator.adoc
+
+[id="troubleshooting-vsphere-configuration_{context}"]
+= Configuration checks for the vSphere problem detector
+
+The following tables identify the configuration checks that are run by the detector.
+Some checks verify the configuration of the cluster.
+Other checks verify the configuration of each node in the cluster.
+
+.Cluster configuration checks
+[options="header",cols="20,50,^15,^15"]
+|===
+|Name
+|Description
+|Periodic
+|Installation
```

Comment:
We don't run any checks during installation in 4.7, we may add some in 4.8. In both cases, IMO it's not really useful to have a column for it.